src/server/webkit/wkInterceptableRequest.ts

```diff
@@ -126,7 +126,24 @@ export class WKInterceptableRequest implements network.RouteDelegate {
       requestStart: timingPayload ? wkMillisToRoundishMillis(timingPayload.requestStart) : -1,
       responseStart: timingPayload ? wkMillisToRoundishMillis(timingPayload.responseStart) : -1,
     };
-    return new network.Response(this.request, responsePayload.status, responsePayload.statusText, headersObjectToArray(responsePayload.headers), timing, getResponseBody);
+
+    const serverIPAddressAndPort: network.IPAddressAndPort = {
+      ipAddress: '',
+      port: -1,
```

Comment:
TODO: This will be actual values once https://github.com/microsoft/playwright/pull/6627 lands.